crates/astria-sequencer/src/sequencer.rs

```diff
@@ -98,3 +104,44 @@ impl Sequencer {
         Ok(())
     }
 }
+
+fn start_grpc_server(storage: &cnidarium::Storage, grpc_addr: std::net::SocketAddr) -> Result<()> {
```

Comment:
IDK if we want to multiplex non-ibc things over this grpc server. Either way, I would prefer to call name it for what it does (serve ibc query requests) rather than for how it does it (being a grpc server):
```suggestion
fn start_ibc_query_server(storage: &cnidarium::Storage, grpc_addr: std::net::SocketAddr) -> Result<()> {
```

Reply:
renamed to `start_ibc_grpc_server` for now, in the future i think we are going to add a lot more stuff to this